src/server/status_server/mod.rs

```diff
@@ -614,13 +623,30 @@ where
                                 Self::get_config(req, &cfg_controller).await
                             }
                             (Method::POST, "/config") => {
-                                Self::update_config(cfg_controller.clone(), req).await
+                                // Check cert since the operation will modify the configuration of TiKV.
+                                if check_cert(security_config, x509) {
```

Comment:
After a second thought, I think the flow should be like this
```
// insert before line 616.
let should_check_cert = match (method, path.as_ref()) {
    (Method::GET, "/metrics") => false,
    (Method::GET, "/status") => false,
    (Method::GET, "/config") => false,
    (Method::GET, "/debug/pprof/profile") => fasle,
    _ => true,
};
if should_check_cert {
   // call check cert
}
// continue the process from line 616.
```

This is to avoid future endpoint missing cert check by mistake.

Reply:
Good point.  I will adopt